pkg/util/validation/limits.go

```diff
@@ -340,6 +348,9 @@ func SetDefaultLimitsForYAMLUnmarshalling(defaults Limits) {
 	defaultLimits = &defaults
 }
 
+// Function to validate Extensions field of Limits.
+var ExtentensionsValidation func(extensions map[string]interface{}) error
```

Comment:
Could this use a getter/setter instead of being exported so that we could validate it's only modified once? Global state makes me nervous.

Reply:
> Could this use a getter/setter instead of being exported so that we could validate it's only modified once?

Yes, we can add check like that.

Reply:
> Yes, we can add check like that.

https://github.com/grafana/mimir/pull/4395/commits/5da5a9a782d4fff86e47bf4d42cb6be626a3d2d0